static/app/components/replays/walker/splitCrumbs.tsx

```diff
@@ -82,11 +82,8 @@ function SingleLinkSegment({
     </Tooltip>
   );
   if (onClick) {
-    return (
-      <Link href="#" onClick={onClick}>
-        {content}
-      </Link>
-    );
+    // TODO(replays): Add a href that deeplinks to `crumb.timestamp`
+    return <Link onClick={onClick}>{content}</Link>;
```

Comment:
Having the `href="#"` in there meant that we could cmd+click and open a new window, pointing at the same url. It wasn't helpful.

We should be able to deeplink to the timestamp, which would be a good use of `href`. Next iteration